src/store/Testdata/sagas.js

```diff
@@ -1,8 +1,10 @@
 import { call, put, takeLatest } from 'redux-saga/effects';
-import { getTestdataSuccess } from './actions';
+import { toast } from 'react-toastify';
```

Comment:
Write utils or service file for this ...

Reply:
If it is not possible now we can make different story for this

Reply:
Yes we can make a different story for this as it uses in multiple files